edx_rest_framework_extensions/auth/bearer/authentication.py

```diff
@@ -81,6 +85,7 @@ def authenticate_credentials(self, token):
         if not user.is_active:
             raise exceptions.AuthenticationFailed('User inactive or deleted.')
 
+        set_custom_metric("BearerAuthentication", "Success")
```

Comment:
Can you move this into the calling method with the out calls to `set_custom_metric`?  Just use a variable to capture the return, then set the metric, and then return.  Thanks.